base/loading.jl

```diff
@@ -263,6 +263,9 @@ const _require_dependencies = Any[] # a list of (path, mtime) tuples that are th
 const _track_dependencies = Ref(false) # set this to true to track the list of file dependencies
 function _include_dependency(_path::AbstractString)
     prev = source_path(nothing)
+    if myid() != 1 && prev === nothing
+        prev = remotecall_fetch(abspath, 1, ".")
+    end
```

Comment:
Looks nice. But is there a reason why the list of dependencies receives the regular path instead of explicitly absolute one?